### DISCUSSION TITLE
Helsinki 2021 - ESP8266 not powered

### DISCUSSION THREAD

**Discussion by Thurgon** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#744] [Q&A]
Hi, first of all thank you for the very comprehensive work. It's admirable what you've done here. 
I have been trying to build this solution for the Helsinki 2021 (6 wire) for some time now. The ESP8266 works for me when powered via the USB port and I connect to it. The firmware is working fine. But when I connect everything on the PCB and plug it into the whirlpool, it doesn't come alive at all. I'm getting pretty desperate because I don't have the tools and probably the knowledge to verify that everything isn't wired correctly. Does anyone see anything wrong? 
![20240703_174406](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/46313903/1ed7a3f0-1213-49d0-87b3-fabd940d2e01)
![20240703_174320](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/46313903/383c6a5a-d3f2-4f38-9b7a-c14ecedc0778)

**Comment by DeanozUK** [comment]
Hi Thurgon,

Just an observation on your PCB, you have to install the resistors R2-R8 (they complete the circuit for the comms) if you are going to use headers H1B & H2B, otherwise if not using the resistors then use headers H1A & H2A.

The resistors and caps are for filtering if there is com issues, mainly 4wire I believe.

Hope this helps.
Dean

**Comment by Thurgon** [comment]
Thanks for the quick reply, but it doesn't work even when H1A & H2A are connected. That's how I had it the first time, but it didn't powered at all, so I thought it was just the absence of capacitors. But the truth is that without resistors it probably can't work like that. I'll add them, but if it didn't just work on H1A & H2A either, it's just not in there. :(

**Comment by visualapproach** [comment]
Your solder joints need to be fixed! I bet that's the problem. Good thing you posted the pictures so we can tell. You need to get heat into both the **pads and the pins, and use flux**! Look at the NodeMCU solder. The tin hugs the pins and the pads. Not a blob on just the pin. The cables need some care too I think. Take my words as constructive critisism to help you out. I'm not hanging you out here.

**Comment by visualapproach** [comment]
And after resoldering the board you should perform the hardware test according to the manual.

**Comment by Thurgon** [comment]
I was a little worried that this would be a problem, but I was hoping it would be enough for conductivity. All right, I'm glad for the constructive criticism. I'll try soldering it again with my shaky hands then. Thanks for your time.
By hardware test you mean /hwtestinfo.html ?

**Reply by visualapproach** [reply]
Exactly 😊
